### PR TITLE
add fetching fields to PropertyDataFetcher

### PR DESCRIPTION
**Pull request by tuukka**
This is for #101

### CODE REVIEW

File: src/main/java/graphql/schema/PropertyDataFetcher.java [hunk 1]
```diff
@@ -48,4 +49,22 @@ private boolean isBooleanProperty(GraphQLOutputType outputType) {
         }
         return false;
     }
+
+    /**
+     * Uses introspection to get the field value.
```
Comment (andimarek) [inline src/main/java/graphql/schema/PropertyDataFetcher.java]: Do you mind removing this comment? Imho there is no real value in it.

**Comment by andimarek** [comment]
Thanks a lot for this PR. Looks good.

**Comment by tuukka** [comment]
I have now removed the unnecessary comment and amended the commit.

**Comment by andimarek** [comment]
Thanks a lot.

**Pull request merged**